projects/ng-boosted/src/lib/o-nav-menu/o-nav-menu.component.html

```diff
@@ -1,3 +1,3 @@
-<button class="nav-link" ngbDropdownToggle [attr.title]="title === 'undefined' ? null : title" [attr.id]="idForLabelledby === 'undefined' ? null : idForLabelledby" [attr.aria-label]="title === 'undefined' ? null : title"
+<button class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" [attr.title]="title === 'undefined' ? null : title" [attr.id]="idForLabelledby === 'undefined' ? null : idForLabelledby" [attr.aria-label]="title === 'undefined' ? null : title"
```

Comment:
No need for `role="button"` here since it's already a `button`.

Reply:
yes, it's done